common/achievement/bucket.ts

```diff
@@ -20,10 +20,7 @@ function createLectureBuckets<T extends ContextMatch | ContextSubcourse>(data: T
             relation: data.relation,
             startTime:
                 measuringType === LectureBucketMeasuringType.start ? moment(lecture.start).subtract(10, 'minutes').toDate() : moment(lecture.start).toDate(),
-            endTime:
-                measuringType === LectureBucketMeasuringType.start
-                    ? moment(lecture.start).add(5, 'minutes').toDate()
-                    : moment(lecture.start).add(lecture.duration, 'minutes').toDate(),
+            endTime: measuringType === LectureBucketMeasuringType.start ? moment(lecture.start).add(5, 'minutes').toDate() : moment(lecture.start).toDate(),
```

Comment:
🤔

Reply:
We removed the duration, in order to prevent further overlapping containers resulting is weird interactions.

This is possible since we agreed on setting the events dates to the lecture start time, when they monitor participation not punctuality.

Reply:
Why only for LectureBucketMeasuringType participation?
Because this is the only event where we add this hack?